src/pages/_app.tsx

```diff
@@ -1,17 +1,15 @@
 import { ChakraProvider } from '@chakra-ui/react'
-import customTheme from 'customTheme'
 import { resetIdCounter } from 'downshift'
 import { DefaultSeo } from 'next-seo'
 import { AppProps } from 'next/app'
 import { useRouter } from 'next/router'
 import { useEffect } from 'react'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { ReactQueryDevtools } from 'react-query/devtools'
-import '@fontsource/open-sans/400.css'
-import '@fontsource/open-sans/700.css'
```

Comment:
@HonkingGoose we don't need to import fonts anymore if they are available through Google Fonts. You'll see how to use them later in this PR.

Reply:
Great! 😄